modules/proc_configuring-annotation-based-tracking-in-multiple-argo-cd-instances.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-annotation-based-tracking-in-multiple-argo-cd-instances_{context}"]
+= Configuring annotation-based tracking in multiple Argo CD instances
+
+You can configure annotation-based tracking in multiple Argo CD instances.
+
+.Pre-requisites
+
+* You have logged in to the {OCP} cluster as an administrator.
+* You have installed the {gitops-title} Operator on your {OCP} cluster.
+
+.Procedure
+
+. Log in to the {OCP} web console. 
+
+. In the *Administrator* perspective of the web console, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to install the user-defined Argo CD instance from the *Project* list.
+
+. Select *{gitops-title}* from the installed Operators list and click the *Argo CD* tab.
+
+. Click *Create ArgoCD* to create two Argo CD instances by using the following YAML snippet.
++
+.Example first Argo CD instance with an annotation label
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: argocd-instance-demo-1 # <1>
+  namespace: argocd-test-demo-1 # <2>
+spec:
+  installationID: "instance-demo-1" # <3>
+  resourceTrackingMethod: "annotation+label"
+----
+<1> Specifies the name of the first Argo CD instance.
+<2> Specifies the namespace used for the first Argo CD instance.
+<3> Specifies the name of the `installationID` object for the first Argo CD instance.
```

Comment:
Is this a "parameter" rather than an "object"? (if so, change in each instance)

Reply:
It is safe to stick to an object because the previous reviewer stated it is an object. Also, I did a little reading online and it is stated that the object can be passed as parameters too.